### ISSUE TITLE
Error after pull from master

### ISSUE THREAD

**Issue by joanna-gil** [shopware/shopware#312]
**Description**  
After pull from master (from Shopware development and platform) had error when try reload page or e.g. clear cache from docker console.

**Possible Solution**  
Find problem in platform/src/Core/Framework/DependencyInjection/message-queue.xml on line 82, 83 and 84. When comment this line, shopware in working fine, but it is still not possible to set colors and fonts in the admin panel in Media > Theme #309 

**Additional context**  
![Screenshot from 2019-11-26 15-26-01](https://user-images.githubusercontent.com/57439261/69641788-1b0e1000-1061-11ea-8ac8-678681efa04f.png)

**Comment by keulinho** [comment]
Hey @joanna-gil 

We've updated our symfony dependencies to version 4.4 today.
Please run `composer update` in your development folder.

After that your system should run smoothly again.

If there are any problems left please let us know.

Kind regards
Jonas

**Comment by joanna-gil** [comment]
@keulinho 
Unfortunately, there was a problem after updating dependency. After updating, I am unable to save anything in the admin panel - `Failed to load resource: the server responded with a status of 500 (Internal Server Error)` appears. I have such a response from the server. Is there any way to change server permissions?
```
{
"errors": [
{
"code": "9",
"status": "401",
"title": "The resource owner or authorization server denied the request.",
"detail": "Missing \"Authorization\" header",
"meta": {
"trace": [
{
"file": "/app/vendor/league/oauth2-server/src/AuthorizationValidators/BearerTokenValidator.php",
"line": 62,
"function": "accessDenied",
"class": "League\\OAuth2\\Server\\Exception\\OAuthServerException",
"type": "::",
"args": [
"Missing \"Authorization\" header"
]
},
{
"file": "/app/vendor/league/oauth2-server/src/ResourceServer.php",
"line": 84,
"function": "validateAuthorization",
"class": "League\\OAuth2\\Server\\AuthorizationValidators\\BearerTokenValidator",
"type": "->",
"args": [
{}
]
},
{
"file": "/app/platform/src/Core/Framework/Api/EventListener/Authentication/ApiAuthenticationListener.php",
"line": 110,
"function": "validateAuthenticatedRequest",
"class": "League\\OAuth2\\Server\\ResourceServer",
"type": "->",
"args": [
{}
]
},
{
"file": "/app/vendor/symfony/event-dispatcher/Debug/WrappedListener.php",
"line": 126,
"function": "validateRequest",
"class": "Shopware\\Core\\Framework\\Api\\EventListener\\Authentication\\ApiAuthenticationListener",
"type": "->",
"args": [
{},
"kernel.controller",
{}
]
},
{
"file": "/app/vendor/symfony/event-dispatcher/EventDispatcher.php",
"line": 264,
"function": "__invoke",
"class": "Symfony\\Component\\EventDispatcher\\Debug\\WrappedListener",
"type": "->",
"args": [
{},
"kernel.controller",
{}
]
},
{
"file": "/app/vendor/symfony/event-dispatcher/EventDispatcher.php",
"line": 239,
"function": "doDispatch",
"class": "Symfony\\Component\\EventDispatcher\\EventDispatcher",
"type": "->",
"args": [
[
{},
{},
{},
{},
{},
{},
{},
{},
{},
{},
{},
{},
{}
],
"kernel.controller",
{}
]
},
{
"file": "/app/vendor/symfony/event-dispatcher/EventDispatcher.php",
"line": 73,
"function": "callListeners",
"class": "Symfony\\Component\\EventDispatcher\\EventDispatcher",
"type": "->",
"args": [
[
{},
{},
{},
{},
{},
{},
{},
{},
{},
{},
{},
{},
{}
],
"kernel.controller",
{}
]
},
{
"file": "/app/platform/src/Core/Framework/Event/BusinessEventDispatcher.php",
"line": 43,
"function": "dispatch",
"class": "Symfony\\Component\\EventDispatcher\\EventDispatcher",
"type": "->",
"args": [
{},
"kernel.controller"
]
},
{
"file": "/app/platform/src/Core/Framework/Event/NestedEventDispatcher.php",
"line": 32,
"function": "dispatch",
"class": "Shopware\\Core\\Framework\\Event\\BusinessEventDispatcher",
"type": "->",
"args": [
{},
"kernel.controller"
]
},
{
"file": "/app/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php",
"line": 168,
"function": "dispatch",
"class": "Shopware\\Core\\Framework\\Event\\NestedEventDispatcher",
"type": "->",
"args": [
{},
"kernel.controller"
]
},
{
"file": "/app/vendor/symfony/http-kernel/HttpKernel.php",
"line": 134,
"function": "dispatch",
"class": "Symfony\\Component\\EventDispatcher\\Debug\\TraceableEventDispatcher",
"type": "->",
"args": [
{},
"kernel.controller"
]
},
{
"file": "/app/vendor/symfony/http-kernel/HttpKernel.php",
"line": 68,
"function": "handleRaw",
"class": "Symfony\\Component\\HttpKernel\\HttpKernel",
"type": "->",
"args": [
{
"attributes": {},
"request": {},
"query": {},
"server": {},
"files": {},
"cookies": {},
"headers": {}
},
1
]
},
{
"file": "/app/vendor/symfony/http-kernel/Kernel.php",
"line": 201,
"function": "handle",
"class": "Symfony\\Component\\HttpKernel\\HttpKernel",
"type": "->",
"args": [
{
"attributes": {},
"request": {},
"query": {},
"server": {},
"files": {},
"cookies": {},
"headers": {}
},
1,
true
]
},
{
"file": "/app/public/index.php",
"line": 92,
"function": "handle",
"class": "Symfony\\Component\\HttpKernel\\Kernel",
"type": "->",
"args": [
{
"attributes": {},
"request": {},
"query": {},
"server": {},
"files": {},
"cookies": {},
"headers": {}
}
]
}
],
"file": "/app/vendor/league/oauth2-server/src/Exception/OAuthServerException.php",
"line": 223
}
}
]
}
```

**Comment by joanna-gil** [comment]
Ok, I just had to build the administration panel again with `./psh.phar administration: bulid`. 
Thank you for your help!

**Comment by joanna-gil** [comment]
After `./psh.phar administration: build` there is still a problem - I can't add new products or change old ones

**Comment by binaermanufaktur** [comment]
@keulinho 
`composer update` logs the following:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - shopware/platform dev-master requires symfony/dependency-injection 4.4.0 -> satisfiable by symfony/dependency-injection[v4.4.0] but these conflict with your requirements or minimum-stability.
    - shopware/platform dev-master requires symfony/dependency-injection 4.4.0 -> satisfiable by symfony/dependency-injection[v4.4.0] but these conflict with your requirements or minimum-stability.
    - shopware/platform dev-master requires symfony/dependency-injection 4.4.0 -> satisfiable by symfony/dependency-injection[v4.4.0] but these conflict with your requirements or minimum-stability.
    - Installation request for shopware/platform dev-master -> satisfiable by shopware/platform[dev-master].

```

**Comment by keulinho** [comment]
@binaermanufaktur 
After updating the platform repo please make also sure to update your development repo before running `composer update`, then the update should succeed.

**Comment by keulinho** [comment]
@joanna-gil 
are there any console output or failing requests?
As the administration works on current master and master of the development template, both on my local setup and in our CI enviroment

**Comment by joanna-gil** [comment]
Yes, there are errors in the console
`PATCH http: // localhost: 8000 / api / v1 / product / 1b80fc22dee743e495b82522ee481b8f 500 (Internal Server Error)` and on the network I have failed the request with the above response. 
I've got it all on a Linux docker

**Comment by keulinho** [comment]
Can you paste the content of the response here?
If it is too big please create a gist with the content

**Comment by joanna-gil** [comment]
https://gist.github.com/joanna-gil/bf21649c7911986a46088e5f45c778be
https://gist.github.com/joanna-gil/c0931693b60ef5f51981d50af0bd8dfa

**Comment by keulinho** [comment]
@joanna-gil 
your DB schema is not up to date.
If your on a dev-enviroment where you work with demo-data that can be dropped and easily recreated it is best to run `./psh.phar install` after each update, this will e.g. apply DB migrations, build the administration and the storefront.

When you don't want to loose all your data you have to run `bin/console database:migrate --all` after each update and rebuild the admin and storefront yourself.

Basically what you have to do to update a local dev enviroment is
```bash
cd development
git pull
cd platform
git pull
cd ..
./psh.phar docker:ssh
composer update
./psh.phar install
```

**Comment by joanna-gil** [comment]
Ok, thanks for the quick reply, now everything works!

**Comment by keulinho** [comment]
Nice thanks for your confirmation @joanna-gil 
Closing this for now

**Comment by binaermanufaktur** [comment]
@keulinho Thanks, everything fine!
